index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>SergSkol Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header class="container header_flex">
+        <div class="logo">My Logo</div>
+        <div class="logo">≡</div>
+    </header>
+    <section class="shapes">
+        <img class="rectangle4" src="./img/Rectangle 4.png" alt="">
+        <img class="elipse3" src="./img/Ellipse 3.png" alt="">
+    </section>
+    <section class="container headline_flex">
+      <br><br><br>
+      <h1 class="headline">I'm Sergiy<br>Glad to see you!</h1>
+      <div class="headline_text">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</div>
```

Comment:
- [ ] Kindly change the `<div>` tag to `<p>` tag. The use of <p> tag in this case would make your tag semantically correct.

Reply:
Thank you! Done!